src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -136,6 +136,66 @@ public static FormulaValue CountRows(IRContext irContext, TableValue[] args)
             return new NumberValue(irContext, count);
         }
 
+        // Count
+        public static FormulaValue Count(IRContext irContext, TableValue[] args)
+        {
+            var arg0 = args[0];
+            var count = 0;
+
+            var errors = new List<ErrorValue>();
+
+            foreach (var row in arg0.Rows)
+            {
+                if (row.Value.GetField(TableValue.ValueName) is ErrorValue error)
```

Comment:
>row.Value [](http://example.com/codeflow?start=20&length=9)

`row.Value` may be null if the row is blank or an error. Examples:

- Count(Table({a:1},{a:2},If(Sqrt(-1)<1,{a:3}),{a:4}) // error row
- Count(Table({a:1},{a:2},If(1<0,{a:3}),{a:4}) // blank row

Reply:
Thanks for making me aware of these scenarios.